include/executor/coredump.h

```diff
@@ -0,0 +1,29 @@
+#include "ast/section.h"
+#include "ast/type.h"
+#include "common/errcode.h"
+#include "common/span.h"
+#include "runtime/instance/memory.h"
+#include "runtime/stackmgr.h"
+
+namespace WasmEdge {
+namespace Coredump {
+
+void generateCoredump(const Runtime::StackManager &StackMgr,
+                      bool ForWasmgdb) noexcept;
+AST::CustomSection createCore();
+AST::CustomSection createCoremodules(
+    Span<const Runtime::Instance::ModuleInstance *const> ModuleInstances);
+AST::CustomSection
+createCorestack(Span<const Runtime::StackManager::Frame> Frames,
+                Span<const Runtime::StackManager::Value> ValueStack,
+                bool ForWasmgdb);
+AST::GlobalSection createGlobals(
```

Comment:
Sorting these functions with the instance order. (i.e. Func, Table, Memory, Global ..)